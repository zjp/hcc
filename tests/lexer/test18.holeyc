a b c d efg '\
boolptr

