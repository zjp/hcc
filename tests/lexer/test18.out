ID:a [1,1]
ID:b [1,3]
ID:c [1,5]
ID:d [1,7]
ID:efg [1,9]
BOOLPTR [2,1]
EOF [4,1]
